docs/docs/overview/supported-tests.mdx

```diff
@@ -0,0 +1,81 @@
+import DocCard from "../../src/theme/DocCard";
+import clsx from "clsx";
+
+# Supported Test Types
+
+Testkube supports running a long list of testing tools and frameworks. You can find the list of supported test types below.
+
+In case your test tool/framework is not currently support, Testkube supports creation of custom Test Types, known as Testkube Container executors. It enables you to run your own container images for executing tests. You can check the guide [here](../test-types/container-executor.md) for more details.
```

Comment:
If your test tool/framework is not currently supported, Testkube supports the creation of custom Test Types, known as Testkube Container executors. It enables you to run your own container images for executing tests. You can check the guide [here](../test-types/container-executor.md) for more details.